packages/app-mobile/root.tsx

```diff
@@ -887,6 +914,7 @@ class AppComponent extends React.Component {
 							value: percent,
 						});
 					}}
+					key={this.state.sideMenuWidth}
```

Comment:
SideMenuWidth is being used to assign the SideMenu component's openMenuOffset prop( which set's the width of the menu ). Last time I checked, I noticed it wasn't causing a rerender. I'll check it again though, I might have missed something.